src/main/scala/org/broadinstitute/hail/utils/richUtils/RichHadoopConfiguration.scala

```diff
@@ -108,15 +108,15 @@ class RichHadoopConfiguration(val hConf: hadoop.conf.Configuration) extends AnyV
   }
 
   def copyMerge(srcFilenames: Array[String], destFilename: String, deleteSource: Boolean = true) {
-
     val destPath = new hadoop.fs.Path(destFilename)
     val destFS = fileSystem(destFilename)
 
     val codecFactory = new CompressionCodecFactory(hConf)
     val codec = Option(codecFactory.getCodec(new hadoop.fs.Path(destFilename)))
     val isBGzip = codec.exists(_.isInstanceOf[BGzipCodec])
 
-    val srcFileStatuses = srcFilenames.flatMap(f => globAndSort(f))
+    val srcFileStatuses = srcFilenames.flatMap(f => glob(f))
```

Comment:
don't glob here, I think.  We've agreed to lift the glob/sort behavior out of copyMerge